core/src/main/java/com/ebner/roomdatabasebackup/core/RoomBackup.kt

```diff
@@ -426,7 +423,7 @@ class RoomBackup {
             return false
         } else if (arrayOfFiles.size > maxFileCount!!) {
             //Sort Array: lastModified
-            Arrays.sort(arrayOfFiles, LastModifiedFileComparator.LASTMODIFIED_COMPARATOR)
+            Arrays.sort(arrayOfFiles, Comparator.comparingLong(File::lastModified))
```

Comment:
I already used Comparator.comparingLong, but this is not compatible with api 21.
See: https://github.com/rafi0101/Android-Room-Database-Backup/commit/fe4e053755496be88872828a4f0e9cdb462ab638#diff-4ec212bd3a4edbeda6ab1cfac646a2e47ed770103f96b1a0bbfad8ed9433697dL420-R417